### ISSUE TITLE
trained result error

### ISSUE THREAD

**Issue by alexander1005** [dspanah/Sensor-Based-Human-Activity-Recognition-DeepConvLSTM-Pytorch#3]
I trained to the code 500 times with lr 0.001 and the result is always below
Epoch: 289/500...Train Loss: 0.1521...Val Loss: 0.7511...Val Acc: 0.8145...F1-Score: 0.8155...

Epoch: 348/500... Train Loss: 0.1344... Val Loss: 0.8809... Val Acc: 0.8039... F1-Score: 0.8052...
Epoch: 349/500... Train Loss: 0.1328... Val Loss: 0.7956... Val Acc: 0.8001... F1-Score: 0.7997...
Epoch: 350/500... Train Loss: 0.1355... Val Loss: 0.8102... Val Acc: 0.8029... F1-Score: 0.8047...
